packages/list-react/documentation/List.mdx

```diff
@@ -74,3 +80,100 @@ Vi kan bruke innrykk under et enkelt listepunkt når det punktet har tilleggsinf
 ## Lister med ikon
 
 Vi bruker lite ikoner i profilen, men det er enkelte unntak. Brukes typisk i en oversikt over hva som er dekket og hva som ikke dekkes.
+
+## Link lister
+
+Vi bruker link-lister for å presentere en liste med lenker. Dette kan være foreksempel være nyttig når du ønsker å lenke innad i en side eller til eksterne sider.
```

Comment:
Hva med "Lenkelister"? Så er vi også konsekvente med "Nummererte/Unummererte lister" lenger opp. Så kan brødteksten være noe á la "Vi bruker komponenten `LinkList` for å presentere[…]"

Reply:
Tror også det ble nevnt i issuet at vi burde trekke frem at det er god praksis å legge lenkelister i en `<nav>`-element med en tittel som sier noe om hva slags navigasjon det er. Kanskje det også kan få plass her før man går inn på de to variantene?